tmk_core/common/chibios/bootloader.c

```diff
@@ -95,6 +95,33 @@ void enter_bootloader_mode_if_requested(void) {
     }
 }
 
+#elif defined(WB32_BOOTLOADER_ADDRESS)  // WB32_BOOTLOADER_DUAL_BANK
```

Comment:
Not a fan of the duplication for WB32_BOOTLOADER_ADDRESS, as the code is virtually the same.

Reply:
Resolved